content/en/docs/ibm/create-cluster-vpc.md

```diff
@@ -0,0 +1,284 @@
+    ID                                          Name                      Status      Subnet CIDR       Addresses     ACL                                                    Public Gateway                             VPC                 Zone         Resource group   
+    0737-27299d09-1d95-4a9d-a491-a6949axxxxxx   my-subnet                 available   10.240.128.0/18   16373/16384   husker-sloping-bee-resize                              my-gateway                                 my-vpc              us-south-3   kubeflow   
+    ```
+
+    If the above list contains the subnet corresponding to your VPC, that can be used to deploy your cluster - make sure
+   you note it's ID.
+   
+    b) To create a new subnet:
+        - List address prefixes and note the CIDR block corresponding to a Zone;
+   in the below example, for Zone: `us-south-3` the CIDR block is : `10.240.128.0/18`.
+   
+    ```shell
+    ibmcloud is vpc-address-prefixes $VPC_ID
+    ```
+    
+    Example output:
+    
+    ```
+    Listing address prefixes of vpc r006-hidden-68cc-4d40-xxxx-4319fa3fxxxx under account IBM as user new@user-email.com...
+    ID                                          Name                                CIDR block        Zone         Has subnets   Is default   Created   
+    r006-xxxxxxxx-4002-46d2-8a4f-f69e7ba3xxxx   rising-rectified-much-brew          10.240.0.0/18     us-south-1   false         true         2021-03-05T14:58:39+05:30   
+    r006-xxxxxxxx-dca9-4321-bb6c-960c4424xxxx   retrial-reversal-pelican-cavalier   10.240.64.0/18    us-south-2   false         true         2021-03-05T14:58:39+05:30   
+    r006-xxxxxxxx-7352-4a46-bfb1-fcbac6cbxxxx   subfloor-certainly-herbal-ajar      10.240.128.0/18   us-south-3   false         true         2021-03-05T14:58:39+05:30  
+    ```
+
+    - Now create a subnet as follows:
+
+    ```shell
+    ibmcloud is subnet-create my-subnet $VPC_ID $CLUSTER_ZONE --ipv4-cidr-block "10.240.128.0/18"
+    ```
+
+    Example output:
+    
+    ```
+    Creating subnet my-subnet in resource group kubeflow under account IBM as user new@user-email.com...
+                          
+    ID                  0737-27299d09-1d95-4a9d-a491-a6949axxxxxx   
+    Name                my-subnet
+    ```
+
+   - Make sure you export the subnet IDs follows:
+
+    ```shell
+    export SUBNET_ID=0737-27299d09-1d95-4a9d-a491-a6949axxxxxx
+    ```
+
+5. Create a `vpc-gen2` based Kubernetes cluster:
+    
+    ```shell
+    ibmcloud ks cluster create vpc-gen2 \
+    --name $CLUSTER_NAME \
+    --zone $CLUSTER_ZONE \
+    --version ${KUBERNETES_VERSION} \
+    --flavor ${WORKER_NODE_FLAVOR} \
+    --vpc-id ${VPC_ID} \
+    --subnet-id ${SUBNET_ID} \
+    --workers 2
+    ```
+
+6. Attach a public gateway
+
+   This step is mandatory for Kubeflow deployment to succeed, because pods need public internet access to download images.
+   
+   First check, if your cluster is already assigned a public gateway:
```

Comment:
```suggestion
    - First, check if your cluster is already assigned a public gateway:
```

(You have sequential steps under "6. Attach a public gateway", so I'm adding "-" to help follow them)